README.md

```diff
@@ -71,20 +71,11 @@ To connect it to slack you need to meet this preconditions:
 > Once your bot is ready for production you should update your commands url to a permanent one.
 > [Heroku](https://duckduckgo.com/?q=flask+on+heroku&t=brave&ia=web) might be a good choice if you are just getting started as it has a generous free tier.
 
-### Does it support new slack Shorcuts?
-Yes, See [examples/shortcuts.py](examples/shortcuts.py) for a self contained example
-
-### And can i use new slack Modals?
-Of course! See [examples/views.py](examples/views.py) for a quick example
-
-### Are interactive actions supported?
-Yes! See [examples/actions.py](examples/actions.py) for a quickstart.
+### Slackify supports Shortcuts, Slash Commands, Events, Actions, Modals, and Block Kit. See the below examples:
```

Comment:
I would prefer something like 
### Slackify Features
- Slash Commands (See [example](examples/commands.py))
- Shortcuts ...
- ...
So one can have a quick overview of examples with links to the relevant code

Reply:
Good idea! I'll get it fixed